Payjoin-spec.md

```diff
@@ -70,6 +70,43 @@ There is no guarantee concerning the ordering of outputs and inputs in either th
 Note: We strongly recommend the sender to submit a PSBT in Base64 format.
 However, we also accept the PSBT or a bitcoin transaction in hex format.
 
+### Optional parameters
+
+When the payjoin sender sends the POST request to the payjoin endpoint, he can optionally specifies the following parameters:
+
+* `v=`, the version number of the payjoin protocol that the client is using. The current version is `1`.
+
+This can be used in the future so the receiver can reject a payjoin if the client is using a version which is not support via an error HTTP 400, `version-unsupported`.
+If not specified, the receiver just assume the sender is using the same version.
+
+If the version of the sender is above the one of the receiver, the receiver should ignore the parameter.
+
+* `feebumpindex=`, the preferred output from which to increase the fee for the added inputs. (default: `-1`)
+
+If the `feebumpindex` is out of bound or pointing on the payment ouptut to the receiver, the receiver should ignore the parameter.
+
+* `noadjustfee=`, a boolean the client specifies if he prefers that the receiver does not bump the fee for the added input. (default: `false`)
+
+Note that if `noadjustfee` is `true`, the sender should create a transaction with RBF disabled, as the original transaction would replace the payjoin transaction.
+
+### Error format
+
+The receiver may send errors to the client, the format is the following:
+
+```json
+{
+    "httpCode": 400,
```

Comment:
The HTTP code here is pointless since this is the body of an HTTP response?

Reply:
yeah